test/processor/rollup.config.js

```diff
@@ -0,0 +1,23 @@
+import scss from '../../index.es.js'
+import postcss from "postcss"
+import autoprefixer from "autoprefixer"
+
+export default {
+  input: '../input.js',
+  output: {
+    file: 'output.js',
+    format: 'esm'
+  },
+  plugins: [
+    scss({
+      processor: css => {
+        return Promise
+          .resolve(
+            postcss([autoprefixer({ overrideBrowserslist: "Edge 18" })])
+              .process(css, { from: undefined })
+          )
+          .then(result => result.css)
+      }
```

Comment:
This seems cumbersome to use, how about including some extra logic to support postcss?

If the plugin checks if `typeof processor.process === 'function'` it can then execute and await that and take the resulting css
```suggestion
      processor: css => postcss([autoprefixer({ overrideBrowserslist: "Edge 18" })])
```

Reply:
Good call out. I guess the reason I didn't consider that is just in case someone wants to use some other tool that isn't postcss, they can do whatever manipulation they want as long as the end result is just the transformed css string. Not sure if that's a likely scenario though. 🤷‍♂️

Reply:
No problem, thanks for the PR 🎉  

I added it here: https://github.com/thgh/rollup-plugin-scss/commit/94cae7f411cb28a078ad6b22c30d6c69de264915#diff-950da4f2d0b2efa15904d9fa37673888R29

Reply:
Indeed! My bundles are very happy. Cheers